pkgs/development/libraries/glib/split-dev-programs.patch

```diff
@@ -12,15 +12,15 @@ index 121e9e6bb..b76fa0188 100644
  )
    # Provide tools for others when we're a subproject and they use the Meson GNOME module
 diff --git a/gio/meson.build b/gio/meson.build
-index 9a9e621b3..12e1d146a 100644
+index 3535788ab..99c3b48d6 100644
 --- a/gio/meson.build
 +++ b/gio/meson.build
-@@ -830,14 +830,15 @@ pkg.generate(libgio,
+@@ -831,14 +831,15 @@ pkg.generate(libgio,
    variables : ['datadir=' + join_paths('${prefix}', get_option('datadir')),
                 'schemasdir=' + join_paths('${datadir}', schemas_subdir),
                 'bindir=' + join_paths('${prefix}', get_option('bindir')),
 +               'devbindir=' + get_option('devbindir'),
-                'giomoduledir=' + giomodulesdir,
+                'giomoduledir=' + pkgconfig_giomodulesdir,
```

Comment:
The change here from upstream was doing a `join_paths('${prefix}...` on the `gio_module_dir` option, which we don't use [0](https://gitlab.gnome.org/GNOME/glib/commit/9f0fc744c6370eafc486ee302085daa7b6f968ba). They renamed the variable at the same time. Updated patch should be fine :+1:

Reply:
Thanks for checking, I come across the same to be sure :heart_decoration: